lib/incoming_mail.rb

```diff
@@ -0,0 +1,110 @@
+module IncomingMail
```

Comment:
The two options for Ruby files in Rails apps are (in my experience) `app/` or `lib/`... what would you suggest? That being said, I think we could be grouping more of the files under `lib/` into subfolders.